plugins/scaffolder-backend-module-cookiecutter/src/actions/fetch/cookiecutter.test.ts

```diff
@@ -47,6 +46,7 @@ jest.mock(
 );
 
 describe('fetch:cookiecutter', () => {
+  const mockDir = createMockDirectory();
```

Comment:
making sure that `mockDir.path` is equal to current `os.tmpdir()` implementation 

```suggestion
  const mockDir = createMockDirectory({ mockOsTmpDir: true });
```